.github/workflows/build.yml

```diff
@@ -111,8 +111,16 @@ jobs:
       # pstuil is an optional package to detect the number of CPU for compiling mmcv
       - name: Install psutil
         run: pip install psutil
-      - name: Build and install
-        run: rm -rf .eggs && pip install -e .
+      - name: Create sdist and untar
+        run: |
+          MMCV_WITH_OPS=1 python setup.py sdist
+          tar zxvf dist/mmcv-full* -C /tmp
```

Comment:
we do not need to untar the package, we can simply use `pip install dist/mmcv-full* `